source/kraken/configuration.cpp

```diff
@@ -59,7 +59,9 @@ po::options_description get_options_description(const boost::optional<std::strin
         ("GENERAL.is_realtime_enabled", po::value<bool>()->default_value(false),
                                         "enable loading of realtime data")
         ("GENERAL.is_realtime_add_enabled", po::value<bool>()->default_value(false),
-                                        "enable loading of realtime data that add stop_times or trips")
+                                        "enable loading of realtime data that add stop_times")
```

Comment:
To me:
is_realtime_enabled == false -> is_realtime_add_enabled = false + enable_realtime_add_trip = false
and
is_realtime_add_enabled == false -> enable_realtime_add_trip = false

So that comment was OK.

Reply:
`is_realtime_enable` is the main flag. It is handled higher. It blocks all realtime functions access.
Later in the code, if we have activate this flag we can have access to `handle_realtime function`, (with a realtime request).
At the beginning of the function `handle_realtime`, we check flag in this order.
1. `enable_realtime_add_trip` and if the realtime request correspond to this type. If it corresponds and the flag is deactivate, we return directly.
2. `is_realtime_add_enabled` and if the realtime request correspond to this type. If it corresponds and the flag is deactivate, we return directly.

https://github.com/CanalTP/navitia/pull/2679/files#diff-a1c1532aa9f44bfd3b8a77e208515e23R537

Then, 
to have an adding trip :
 `is_realtime_enabled == true + enable_realtime_add_trip = true`
to handle added stop time :
 `is_realtime_enabled == true + enable_realtime_add_enabled = true`
if  `is_realtime_enabled == false`, we handle anything

Am I clear ? One main flag and one flag by feature (add trip, add stop time). I think it is a simple rule and it works.
We can talk about it with the others team boys if it necessary.

Reply:
**To sum up** (After stand up)

we'll combine the logical flags new rules are
- **is_realtime_enabled** is the main flag, it can be block all realtime access
- **is_realtime_add_stop_enabled** the stronger than **is_realtime_add_trip_enabled**
  If `is_realtime_add_stop_enabled == false` we can't active the _adding trip option_
  so,  `is_realtime_add_trip_enabled == false` that doesn't block the _adding stop time option_